KIPs/kip-7-fungible_token.md

```diff
@@ -119,6 +120,79 @@ interface IKIP7 {
     /// @param recipient The owner will receive the tokens.
     /// @param amount The token amount will be transferred.
     function transferFrom(address sender, address recipient, uint256 amount) external returns (bool);
+    
+    /// @notice Moves `amount` tokens from the caller's account to `recipient`.
+    /// @dev Throws if the message caller's balance does not have enough tokens to spend.
```

Comment:
I like these tags. However, they don't have consistency with previously existing functions. 
Why don't you add `@notice` tag to previously existing functions?

Reply:
OK I will add `@notice` to all function. Thanks.